radsat-sk/operation/subsystems/RAntenna.c

```diff
@@ -26,7 +26,8 @@ static int antennaInitialized = 0;
 /** Track Number of times the antenna attempted to deploy*/
 static int antennaDeploymentAttempts = 0;
 
-
+/** Index of the Antenna*/
+#define ANTENNA_INDEX 0
 /***************************************************************************************************
```

Comment:
Whitespace issues:
- Add a space after the word `Antenna` of the comment (between `a` and `*`)
- Add 2 newlines between the `#define` and the section separator (on line 30)